function_calling.ipynb

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    upstage_api_key = "FmfN0JETjndtc4Xz9awTtLiLUa7DmCsc"
</pre><p>Please remove this key from the commit and DELETE the key from the console as well. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UpstageAI/cookbook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>